src/Components/App/App.js

```diff
@@ -78,10 +78,12 @@ function App() {
         <Nav />
         {showError && <ErrorPage error={error} />} 
         <Routes>
-          <Route path="/" element={<HomePage error={error}  userImage={userImage} onImageUpload={setUserImage} renderMushroomCards={renderMushroomCards}/>} /> 
-          <Route path="/error" element={<ErrorPage error={error} />} /> 
-          <Route path="/myfieldguide" element={<FieldGuide error={error} renderMushroomCards={renderMushroomCards} />} />
-          <Route path="/details/:id" element={<DetailsPage />} />
+          <Route path="/" element={<HomePage error={error} userImage={userImage} onImageUpload={setUserImage} renderMushroomCards={renderMushroomCards} />} />
+          <Route path="/error" element={<ErrorPage error={error} />} />
+          <Route path="/myfieldguide" element={<FieldGuide error={error} />}>
+            <Route path="details/:id" element={<DetailsPage />} />
```

Comment:
I'm so glad we figured this out! Amazing that something so simple `../` can make such a huge difference. What a great learning opportunity!